src/js/tracks/text-track-settings.js

```diff
@@ -22,36 +23,127 @@ const LOCAL_STORAGE_KEY = 'vjs-text-track-settings';
 //   The selector used to find the associated <select> element.
 const selectConfigs = {
   backgroundColor: {
-    selector: '.vjs-bg-color > select'
+    selector: '.vjs-bg-color > select',
+    id: 'captions-background-color-%s',
+    label: 'Color',
+    options: [
+      ['#000', 'Black'],
+      ['#FFF', 'White'],
+      ['#F00', 'Red'],
+      ['#0F0', 'Green'],
+      ['#00F', 'Blue'],
+      ['#FF0', 'Yellow'],
+      ['#F0F', 'Magenta'],
+      ['#0FF', 'Cyan']
+    ]
   },
+
   backgroundOpacity: {
-    selector: '.vjs-bg-opacity > select'
+    selector: '.vjs-bg-opacity > select',
+    id: 'captions-background-opacity-%s',
+    label: 'Transparency',
+    options: [
+      ['1', 'Opaque'],
+      ['0.5', 'Semi-Transparent'],
+      ['0', 'Transparent']
+    ]
   },
+
   color: {
-    selector: '.vjs-fg-color > select'
+    selector: '.vjs-fg-color > select',
+    id: 'captions-foreground-color-%s',
+    label: 'Color',
+    options: [
+      ['#FFF', 'White'],
```

Comment:
We should define the colors at the top once as a const, since we are using them in two places